packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -80,8 +80,8 @@ export function generateRuntimeOptions(
         enableOfflineLoad: [undefined],
         flushMode: [undefined],
         compressionOptions: [{ minimumBatchSizeInBytes: 500, compressionAlgorithm: CompressionAlgorithms.lz4 }],
-        maxBatchSizeInBytes: [undefined],
-        chunkSizeInBytes: [undefined],
+        maxBatchSizeInBytes: [972800],
+        chunkSizeInBytes: [614400],
```

Comment:
What's the rationale for these numbers in particular?

Reply:
I'll add some comments for this.

If unspecified, `maxBatchSizeInBytes` is already `972800`. This is just making it explicit. The actual hard max is 1MB but we want a margin of safety to account for stringification overhead, metadata, etc..

`chunkSizeInBytes` is a trade-off between large few ops vs small many ops. The latter _may_ lead to some throttling. 600k is a reasonable middle ground IMHO. It can be up to `972800`